server/report_builder/README.md

```diff
@@ -197,10 +197,11 @@ This can be implemented in the report by adding the following translation functi
 By default,
 
 Where the letters are short hand for the following:
+
 - t for translate
   The name of the function
-- k for key 
-  This is the locale key as is used in front end translations. 
+- k for key
+  This is the locale key as is used in front end translations.
 - n for namespace
   The file namespace where the translation key is. The .json exention is automattically added ie catalogue (which refers to the catalogue.json namespace).
```

Comment:
```suggestion
  The file namespace where the translation key is. The .json extention is automatically added ie catalogue (which refers to the catalogue.json namespace).
```
But maybe we can remove this line now that we just have `common.json`

Reply:
Have updated to to specify that we aren't using namespace anymore, but functionality is retained (in case we use customer specific namespaces in future)